.github/workflows/devcontainer.yml

```
@@ -0,0 +1,137 @@
+          else
+            echo "exists=true" >> $GITHUB_OUTPUT
+          fi
+      # repository owner to lowercase for Docker image naming, as devcontainers/ci does not safeguard
+      - name: Get lowercase repository owner
+        id: repo_owner
+        run: echo "owner_lc=$(echo '${{ github.repository_owner }}' | tr '[:upper:]' '[:lower:]')" >> $GITHUB_OUTPUT
+      
+      - name: Get Image Name
+        if: steps.check.outputs.exists == 'true'
+        id: image_name
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2086:info:24:34: Double quote to prevent globbing and word splitting

(shellcheck)